INSTALL.md

```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
 
  **Once you see the screen above, you have successfully setup LibreEHR! Congratulations!**
 
+## On Mac With MAMP (For Local Testing)
+Apache can be used on Mac to install LibreHealth. However, if you have installation problems, you can use MAMP.
+
+#### Setup
+Install MAMP version 3.5.6 or under (Newer versions come with PHP 7.1 which is not compatible with LibreHealthEHR. If youdo install a newer version, make sure to use PHP versions under 7.1). You can find older versions.
+
+#### Note: PHP must not be version 7.1 or above. Version 7.0 and 5.6 can both be used.
+
+Firstly, download [MAMP](https://mamp.info/en/) version 3.5.6. Currently, we do not support PHP versions 7.1. MAMP 3.5.6 comes with PHP 7.0 so you can use it without much configuration. Older Versions (including version 3.5.6) can be found [here](https://mamp.info/en/downloads/older-versions/)
+
```

Comment:
Let's ensure that people use a proper version -- which is not close to EOL -- ensure that _PHP 7.1 or better is used.

Reply:
sure @punwai please see comment thanks @robbyoconnor

Reply:
@robbyoconnor  @muarachmann Is there support for 7.1 ++ yet? Or is it going to be implemented soon